internal/apis/core/v1alpha3/cluster_types.go

```diff
@@ -135,7 +135,11 @@ type ClusterStatus struct {
 	// +optional
 	ControlPlaneInitialized bool `json:"controlPlaneInitialized"`
 
-	// ControlPlaneReady defines if the control plane is ready.
+	// ControlPlaneReady denotes if the control plane API Server became ready during initial provisioning
```

Comment:
Please only modify the godoc in https://github.com/kubernetes-sigs/cluster-api/blob/main/api/v1beta1/cluster_types.go (and rollback the changes to v1alpha3 & v1alpha4)

Reply:
Updated with latest commit.